### ISSUE TITLE
Impossible to do a composer install when elastic apm is enabled

### ISSUE THREAD

**Issue by julien-maurel** [elastic/apm-agent-php#876]
**Describe the bug**
Impossible to do a composer install when elastic apm is enabled

**To Reproduce**
Use docker php image : docker run -it --rm php:8.1.16-bullseye bash
In docker container :
```
curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer
apt-get update
apt-get install zip procps
composer require --dev phpunit/phpunit 

# This install is ok
rm -rf vendor/ ~/.composer/cache/
composer install 

# Install APM
curl -sSLO https://github.com/elastic/apm-agent-php/releases/download/v1.8.0/apm-agent-php_1.8.0_all.deb
dpkg -i apm-agent-php_1.8.0_all.deb

# This install is ko, php keep running
rm -rf vendor/ ~/.composer/cache/
composer install
```

By running a `ps auxf` in another terminal, we ca see a lot of php process pending
```
root@2689d3ad1019:/# ps auxf
USER         PID %CPU %MEM    VSZ   RSS TTY      STAT START   TIME COMMAND
root         816  2.0  0.0   4164  3356 pts/1    Ss   11:37   0:00 bash
root         822  0.0  0.0   6760  2864 pts/1    R+   11:37   0:00  \_ ps auxf
root           1  0.0  0.0   4164  2328 pts/0    Ss   11:22   0:00 bash
root         778 90.1  0.3 667016 56808 pts/0    R+   11:36   0:11 php /usr/local/bin/composer install
root         806  0.0  0.2 667016 36004 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         807  0.0  0.2 667016 36004 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         808  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         809  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         810  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         811  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         812  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         813  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         814  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
root         815  0.0  0.2 667016 36008 pts/0    S+   11:36   0:00  \_ php /usr/local/bin/composer install
```

I tried to install older version of apm, 1.7.2 is broken too, 1.7.1 is working...
It's important to remove composer vendor/cache before install because it's vendor downloading that generate this behavior :)

**Expected behavior**
Composer install should work :)

**Comment by Lorrhanl** [comment]
I also tried version 1.8.1 and had the same problem.

**Comment by Zorinik** [comment]
Same problem here, major issue as we can't upgrade to PHP 8.2 due to this. I just tried the just-released 1.8.3 with same results
